test/models/todo_list_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```

Comment:
what is the purpose of this file? if you haven't written any test cases. Either remove the file or write proper test cases.
Similarly for other files as well